core-services/pipeline-controller/config.yaml

```diff
@@ -2,3 +2,4 @@ orgs:
   - org: openshift
     repos:
       - ci-tools
+      - dpu-operator
```

Comment:
That's because sriov network operator doesn't use this special pipeline-controller mechanism. We need to add the dpu-operator repo to the pipeline-controller config in order to enable this functionality (lightweight and heavyweight tests).

Reply:
I don't see it in the docs... https://docs.ci.openshift.org/docs/how-tos/creating-a-pipeline/ .

Reply:
This is correct. This could be an undocumented portion... however a member of the test platform team (@jmguzik) told me to add it here in order to lightweight/heavyweight tests.